src/_scss/pages/modals/all.scss

```diff
@@ -1,5 +1,11 @@
+// TODO: [DEV-5671] Tech Debt: USAspending Shared Styles and Reducing SCSS Bundle
+// Clean up these modal styles so they're more shared.
+// covid19 and redirect modal share the same classnames.
+
 @import "./extraModal/extraModal";
 @import "./fullDownload/fullDownload";
 @import "./bulkDownload/bulkDownload";
-@import "./redirect/redirectModal";
-@import "./recipient/recipient";
\ No newline at end of file
+@import "./recipient/recipient";
+@import "./redirect/redirect";
+@import "../../components/baseModal";
```

Comment:
We may want to import baseModal first to facilitate overriding styles in files for specific modals?

Reply:
We should... but it breaks the other styles. It actually has to be this way. Definitely tech debt.